shell/platform/android/io/flutter/plugin/common/EventChannel.java

```diff
@@ -88,7 +88,7 @@ public void setStreamHandler(final StreamHandler handler) {
   }
 
   /**
-   * Handler of stream setup and tear-down requests.
+   * Handler of stream setup and teardown requests.
```

Comment:
Should probably be "set up", but I can see an agreement for either being correct.